keras/ops/nn_test.py

```diff
@@ -1750,6 +1750,22 @@ def test_batch_normalization(self):
         )
         self.assertEqual(tuple(output.shape), (2, 3, 3, 5))
 
+    def test_ctc_loss(self):
```

Comment:
Either skip the JAX backend in this test, or implement the ctc_loss op for JAX as well

Reply:
I skipped it for now, thanks